code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -11,16 +11,28 @@
 	door_anim_time = 0
 
 /obj/structure/closet/crate/necropolis/tendril
-	desc = "It's watching you suspiciously."
+	desc = "It's watching you suspiciously. You need a skeleton key to open it."
```

Comment:
Set the you need a key part in the examine, so when you examine it after its unlocked it doesn't show that.

Reply:
![image](https://user-images.githubusercontent.com/3241376/117605807-2bfbf580-b11e-11eb-91c8-d53f08697494.png)

![image](https://user-images.githubusercontent.com/3241376/117605892-58177680-b11e-11eb-83e0-4cd7b92e57a5.png)

![image](https://user-images.githubusercontent.com/3241376/117605901-5c439400-b11e-11eb-8008-f3ce4c72a52e.png)